tutorials/spanner-oc/index.md

```diff
@@ -0,0 +1,141 @@
+3. Instance configuration `regional-us-central1`
+4. Node count of 1 (`node_count` corresponds to the amount of serving and storage resources available to databases in the instance.)
+
+## Create and configure a Compute Engine VM
+
+Create a Compute Engine VM with the necessary security scopes by running the following commands from Cloud Shell. 
+
+    gcloud compute instances create trace-client  --zone=us-central1-c \
+    --scopes="https://www.googleapis.com/auth/spanner.admin.table,\
+    https://www.googleapis.com/auth/spanner.data,\
+    https://www.googleapis.com/auth/logging.write,\
+    https://www.googleapis.com/auth/monitoring.write,\
+    https://www.googleapis.com/auth/trace.append"
+
+## Deploying the Java application
+
+In this section, you download the Java application containing the instrumented code, make the necessary modifications to reflect your environment and then run it.
+
+1.  Log in to the VM
+
+        gcloud compute ssh trace-client --zone=us-central1-c
+        
+1.  Run the following command to install Git, the Java 8 JDK, and Maven:
+
+        sudo apt-get install git openjdk-8-jdk maven -y
+
+1.  Clone the source repository for this tutorial:
+
+        git clone https://github.com/GoogleCloudPlatform/community.git
+        
+Update the Java application with some configuration specific to your project.
+
+1.  Navigate to the folder containing the Java source:
+
+        cd community/tutorials/spanner-oc/java/
+
+1.  Configure the environment variable for the application code to use the `spanner-oc` Spanner instance and `demo-1` as database id:
+
+        export INSTANCE_ID=spanner-oc
+        export DATABASE_ID=demo-1
+        
+1.  Run the following maven commands to build and run the program:
+
+        mvn package -DskipTests --quiet
+        mvn exec:java -Dexec.mainClass=com.example.spanner.App --quiet
+
+## Viewing traces in Stackdriver Trace UI
+
+As soon as the code ran, it started exporting to Stackdriver.
+
+1.  Go to the [Stackdriver Trace console](https://cloud.google.com/console/traces).
+
+1.  If you select **create-players**, you'll be taken to a view 
+that shows more information about the call chain, along with other useful 
+information such as lower-level tracing and operation-level call latencies.
+
+    Drilling down to a particular data point in time yields a trace with a root span and children spans shown below:
+
+    ![Screenshot](./images/trace-timeline.png)
+
+1.  The power of visualization helps teams perform root cause analyses, post-mortems, bisect bad changes, figure out what needs to be optimized.
+
+1.  Please note that for this demo we are always sampling and that is very high. More realistically perhaps tracing 1 in 10,000 might be suffice.
+
+## Viewing metrics in Stackdriver Metrics Explorer UI
+
+The tutorial also demonstrates the recording of application metrics. Latency and transaction counts will be measured by the application code and recorded.
```

Comment:
nit:s/Latency and transaction counts/Transaction Latency and Transaction Count/

Reply:
Done